Bio/Align/__init__.py

```diff
@@ -932,6 +933,82 @@ def sort(self, key=None, reverse=False):
         else:
             self._records.sort(key=key, reverse=reverse)
 
+    @property
+    def substitutions(self):
+        """Return an Array with the number of substitutions of letters in the alignment.
+
+        As an example, consider a multiple sequence alignment of three DNA sequences::
```

Comment:
Try reducing the double colon to a single colon - the following content is a doctext, not a literal block. That should I think fix this:

```
$ flake8
./Bio/Align/__init__.py:951:1: RST399 Inconsistent literal block quoting.
```

Reply:
Thanks, that did the trick.